tools/afl-fuzzer/platform/shim.h

```diff
@@ -0,0 +1,157 @@
+/*
```

Comment:
It probably makse sense to include the FW version of these headers

Reply:
I think we should be including this from the core project